lib/Flow/Operation.php

```diff
@@ -44,30 +44,13 @@
 use function json_encode;
 
 class Operation implements IOperation {
-
-	/** @var IL10N */
-	private $l;
-	/** @var IURLGenerator */
-	private $urlGenerator;
-	/** @var IManager */
-	private $notificationManager;
-	/** @var IUserSession */
-	private $userSession;
-	/** @var LoggerInterface */
-	private $logger;
-
 	public function __construct(
-		IL10N $l,
-		IURLGenerator $urlGenerator,
-		IManager $notificationManager,
-		IUserSession $userSession,
-		LoggerInterface $logger
+		private IL10N $l,
+		private IURLGenerator $urlGenerator,
+		private IManager $notificationManager,
+		private IUserSession $userSession,
+		private LoggerInterface $logger
```

Comment:
```suggestion
		private LoggerInterface $logger,
```

For easier future, also allowed in php 8.0+